src/app/plugins/marketplace.plugin.ts

```diff
@@ -90,4 +72,35 @@ export class MarketPlacePlugin extends Plugin {
       await message.reply(embed);
     }
   }
+
+  private async _fetchMessages(message: IMessage, limitParam: number) {
+    let i: number;
+    let last_id = message.id;
+    const itemsForSale: String[] = [];
+
+    for (i = 0; i < limitParam / 100; i++) {
+      const config = { limit: 100, before: last_id };
+      const batch = await message.channel.fetchMessages(config);
+
+      batch.forEach((msg) => {
+        const { content } = msg;
+
+        //If content does not contains either element of array then stop.
+        if (
+          ![this._NEW_LISTING_PREFIX, this._NEW_ALIAS_PREFIX].some((el) => content.includes(el))
+        ) {
+          return;
+        }
+
+        let [, item] = content.split('add');
```

Comment:
const

Reply:
Cant use const, there is a reassign on line 98